packages/ui/components/card/Card.tsx

```diff
@@ -142,17 +143,14 @@ export function Card({
       {variant === "SidebarCard" && (
         <div className="mt-2 flex items-center justify-between">
           {learnMore && (
-            <Link
+            <LinkComponent
               href={learnMore.href}
               onClick={actionButton?.onClick}
               target="_blank"
               rel="noreferrer"
-              className="text-xs font-medium"
-              // NextJS thinks this is an internal link and tries to prefetch it (also on hover), which fails
-              // TODO: Extract this into a constant to handle /docs & /developer as well
-              prefetch={!learnMore.href.startsWith("https://cal.com/blog")}>
```

Comment:
fixes prefetch={true} is depreceated warning

https://nextjs.org/docs/messages/prefetch-true-deprecated